js/base/Exchange.js

```diff
@@ -2226,6 +2226,16 @@ module.exports = class Exchange {
         } else {
             throw new NotSupported (this.id + ' fetchMarketLeverageTiers() is not supported yet');
         }
+    }
 
+    parseOpenInterests (response, symbol, since, limit) {
+        const interests = [];
+        for (let i = 0; i < response.length; i++) {
+            const entry = response[i];
+            const interest = this.parseOpenInterest (entry);
```

Comment:
Let's accept a `market` instead of a `symbol` in parseOpenInterests and let's also propagate the `market` to `parseOpenInterest` as the second argument like we do with the other parsers, to keep it uniform.